pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -767,6 +795,30 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 		domain.Spec.Devices.Rng = newRng
 	}
 
+	//usb controller is turned on, only when user specify input device with usb bus,
+	//otherwise it is turned off
+	if usbDeviceExists := isUSBDevicePresent(vmi); !usbDeviceExists {
+		// disable usb controller
+		domain.Spec.Devices.Controllers = append(domain.Spec.Devices.Controllers, Controller{
```

Comment:
Perhaps we shouldn't manage the controllers here and instead just enable the usb controller by default.
I think @michalskrivanek has a point[1] arguing that it's best to keep the controller enabled by default.
This will also simplify the logic here and will remove the need for `isUSBDevicePresent` method.

@MarSik what do you think?

[1] https://github.com/kubevirt/kubevirt/pull/2008#issuecomment-462688983

Reply:
@MarSik, @michalskrivanek, @fabiand ^^^^^^^
should i enable usb controller by default?

Reply:
@vladikr I spoke with @MarSik and he said he is ok with this approach, as it is in this PR. In Kubevirt it was disabled, because in ovirt it was disabled as well, but in ovirt it was not disabled because of measured performance issues. So if you would like to have it enabled by default i will change it.

Reply:
Yeah, I believe the current approach is as good as enabling usb by default. The reason oVirt disabled USB was to prevent users from playing with hotplug and spice passthrough features. The same might apply to kubevirt when we add those features.

So I do not see any strong reason to rewrite this again.